.circleci/config.yml

```diff
@@ -85,16 +85,16 @@ jobs:
     - CASS_DRIVER_NO_EXTENSIONS: true
     - CASS_DRIVER_NO_CYTHON: true
     - CASSANDRA_SKIP_SYNC: true
-    - DTEST_REPO: git://github.com/apache/cassandra-dtest.git
-    - DTEST_BRANCH: master
+    - DTEST_REPO: git://github.com/jasonstack/cassandra-dtest.git
+    - DTEST_BRANCH: CASSANDRA-15666
```

Comment:
redirect circle-ci to use custom dtest branch, test pending: https://circleci.com/workflow-run/85838845-bef8-4015-aff9-8a2246c348c2